tox.ini

```diff
@@ -10,6 +10,7 @@ envlist =
 deps =
   pytest
   pdbpp
+  defusedxml
```

Comment:
Why is this new dependency required?

Reply:
Oh, I saw you added it in the last commit because tests were failing on latest versions.